### PR TITLE
first commit for movie app:  movie card, rating, comments, styling 

### PR DESCRIPTION
**Pull request by CiprianP84**

### CODE REVIEW

File: movieDetails.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
+import { moviesDatabase } from "./movieStorage.js";
+
+// Create movie cards
+function createMovieCard(movie) {
```
Comment (vdaguenet) [inline movieDetails.js]: This function is quite long. You could split it into multiple parts:
1. Display the movie info
2. Create the star rating component
3. Create the comment component

File: movieDetails.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+  popupContainer.append(headerContainer, commentsSection, commentInput, submitButton);
+  popupBackground.appendChild(popupContainer);
+  document.body.appendChild(popupBackground);
+
+  closeIcon.addEventListener("click", function () {
+    document.body.removeChild(popupBackground);
+  });
+
+  submitButton.addEventListener("click", function () {
```
Comment (vdaguenet) [inline movieDetails.js]: This could be extracted in another function to not make `openCommentsPopup` so long.

File: movieDetails.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+      star.classList.remove("fa-regular");
+    } else {
+      star.classList.add("fa-regular");
+      star.classList.remove("fa-solid");
+    }
+  });
+}
+
+// Popup and comment functionality 
+function openCommentsPopup(movie) {
```
Comment (vdaguenet) [inline movieDetails.js]: Same comment as my previous one. You can split this function into smaller functions to make your code more readable.

**Review by vdaguenet — APPROVED** [review]
It's an excellent work, Ciprian! I now want to subscribe to TazFlix 😉 

Be mindful of the length of your functions. It's easier to understand and debug small functions than functions with 80+ lines of code.

**Comment by CiprianP84** [comment]
Thanks for the feedback. I tried a different approach now that helped a lot with the length of my functions and repetition. 
What do you think?
I can still see some places that could be improved so I will work on them for the future submissions.

**Comment by vdaguenet** [comment]
> Thanks for the feedback. I tried a different approach now that helped a lot with the length of my functions and repetition. What do you think? I can still see some places that could be improved so I will work on them for the future submissions.

Nice changes! It's easier to follow your logic now

**Review by vdaguenet — APPROVED** [review]

**Pull request merged**